### DISCUSSION TITLE
Swiss not Loading

### DISCUSSION THREAD

**Discussion by Akurie** [emukidid/swiss-gc#751] [Q&A]
Good Day, I am running with a pi pico with .2v picoboot.uf2 latest Swiss loaded to a 126 Micro SD (FAT32) plugged into a GC2SD in slot A.  I followed the instruction to the fullest extent and I keep Getting the CG IPL not Swiss.  I am sure I might be missing something.  Looking for guidance.

**Comment by Colrut** [comment]
When this happens it is usually a connection issue on the GC or Pico board.  Check all the solder points carefully to make sure that none are bridged to other contacts.  I have gotten tripped up where the heat sink presses on the IPL chip and the wires coming off of it.  I run the wire behind/around the chip rather than over.  On one build I put a bit of heat sink over it as well to relieve pressure and insulate.

Good luck

![PXL_20230126_100652960](https://user-images.githubusercontent.com/83892045/214896447-7c838f0a-58be-47bc-a47e-153fdbd57eb1.jpg)

**Comment by Akurie** [comment]
Thank you for the advice, I will be looking into it soon.
